AUTHORS

```diff
@@ -0,0 +1,46 @@
+# This is the official list of Syncthing-Android authors for copyright purposes.
+# The format is:
+#
+#    Name Name Name (nickname) <email1@example.com> <email2@example.com>
+#
+
+Adam Piggott (ProactiveServices) <aD@simplypeachy.co.uk> <simplypeachy@users.noreply.github.com> 
+Alex Saveau <SUPERCILEX@users.noreply.github.com>
+Alexander Lochmann <flipr3vere@googlemail.com>
+Alexander Sosedkin <monk@unboiled.info>
+Andrey D (scienmind) <scintertech@cryptolab.net>
+anrp <anrp@anrp.net>
+Audrius Butkevicius (AudriusButkevicius) <audrius.butkevicius@gmail.com>
+Ben Holden-Crowther <benhc123@users.noreply.github.com>
+Brian Lin <brianhlin@gmail.com>
+Carsten H <moter8@gmail.com>
+Cathryne Linenweaver (Cathryne) <cathryne.linenweaver@gmail.com> <Cathryne@users.noreply.github.com>
+Cedric Staniewski <cedric@gmx.ca>
+Charles Collins <super9mega@gmail.com>
+Chris McCormick <chris@mccormick.cx>
+Cristian Beskid <schizosfera@gmail.com>
+Faisal Hameed <faisal.hameed@devfactory.com>
+Felix Ableitner <me@nutomic.com>
+FlashSystems <developer@flashsystems.de>
+George Venios <veniosg@gmail.com>
+Jakob Borg (calmh) <jakob@nym.se> <jakob@kastelo.net>
+Jessie Chatham Spencer <isai@protonmail.com>
+JohnVeness <john.veness.github@pelago.org.uk>
+kDot <k-Laus@gmx.net>
+klaus <k-Laus@gmx.net>
```

Comment:
There is a ton of duplicates with same email.
Also, I see no purpose of this, just downsides of having to maintain this pointless file

Reply:
-- deleted as discussion is going on in the forum --